cmd/snap/cmd_snap_op.go

```diff
@@ -94,6 +94,10 @@ have developer access to the snap, either directly or through the
 store's collaboration feature, and to be logged in (see 'snap help login').
 
 Note a later refresh will typically undo a revision override.
+
+Using --hold without specifying any snaps will hold all snap refreshes. Holds
+are only applied to general refreshes (including auto-refreshes), but snaps can
+still be refreshed in specific refreshes (i.e, 'snap refresh snap-a snap-b').
```

Comment:
Holds (--hold) specified for snaps are only applied to general refreshes ('snap refresh' without specifying snaps and auto-refreshes), but snaps can still be refreshed in specific refreshes (i.e, 'snap refresh snap-a snap-b'). Using --hold without specifying any snaps will hold auto-refreshes only for all snaps.